questionpy_server/api/routes.py

```diff
@@ -55,9 +54,9 @@ async def post_options(request: web.Request, package: Package, question_state: O
 
     package_path = await package.get_path()
     async with qpyserver.worker_pool.get_worker(package_path, 0, data.context) as worker:
-        definition = await worker.get_options_form_definition()
+        definition, form_data = await worker.get_options_form(question_state)
 
-    return json_response(data=definition)
+    return json_response(data=QuestionEditFormResponse(definition=definition, form_data=form_data))
```

Comment:
In der` qppe.yaml` sollte das auch noch angepasst werden. Ich glaube da fehlt sogar ein entsprechendes `Component`, was die `QuestionEditFormResponse` abbildet.